events/c_2_project_and_community.toml

```diff
@@ -19,7 +19,7 @@ date = "2022-07-14"
 days = 1
 
 # the event times (shows up in the event card)
-times = "09:00 - 17:00"
+times = "09:00 - 12:00"
```

Comment:
@mishmosh do you have any objection to making yourself the `dri` above?

on another note: I think targeting a half day would be fantastic.

Reply:
sounds good. i will be scouting aggressively for a replacement dri for 2023.